jOOQ-meta/src/main/resources/xsd/jooq-codegen-3.11.0.xsd

```diff
@@ -1033,6 +1037,15 @@ This flag is ignored in the commercial Java 6 distribution of jOOQ 3.9+ ]]></jxb
     </all>
   </complexType>
 
+
+  <simpleType name="JpaVersion">
+    <restriction base="string">
+      <enumeration value="V_2_0"/>
+      <enumeration value="V_2_1"/>
+      <enumeration value="V_2_2"/>
```

Comment:
I understand that you've introduced this enumeration syntax for XJC to pick up the values as actual enum constants, which wouldn't be possible if we were using `2.0`, `2.1`, and `2.2` instead. But still, the configuration should be XML / declarative first, and programmatic second, so I'd really prefer this to be simple strings that correspond to the JPA version scheme on a 1:1 basis.